src/components/Recipes.js

```diff
@@ -0,0 +1,146 @@
+import { v4 as uuid } from 'uuid';
+import List from '@mui/material/List';
+import Divider from '@mui/material/Divider';
+import Typography from '@mui/material/Typography';
+import React, { useEffect, useState } from 'react';
+import * as recipesClient from '../api/recipes';
```

Comment:
I think it's better to import every single export separately than to import everything with `*`.

Reply:
The thing with this is that the functions' names inside recipesClient would conflict with methods in the Recipes.jsx file. Both files have getRecipes, addRecipe, editRecipe and removeRecipe functions which deal with the same entities (recipes) and the same actions (get, add, edit, remove) but in different layers (api connector vs ui controller) so we are left with two choices: either rename functions to explicitly specify the layer they belong to or consume the rest methods through the recipesClient object, which I think is a little bit more legible. What do you think about this?

Reply:
True, it makes sense in this case when you want all of the exports from the API. If we were to use only one function, we could do something like: 
```typescript
import { getRecipes as getRecipesAPI } from '../api/recipes/'
```